test/server/cards/03_TWI/units/ConfederateCourier.spec.ts

```diff
@@ -0,0 +1,21 @@
+describe('Confederate Courier', function () {
+    integration(function (contextRef) {
+        it('ability should create a Battle Droid token when defeated', function () {
```

Comment:
nit: should start with `Confederate Courier\'s ability should...`. Same for other tests